src/Tasks/WeekPlanner/WeekPlanner.java

```diff
@@ -0,0 +1,69 @@
+package Tasks.WeekPlanner;
+
+import java.util.Scanner;
+
+public class WeekPlanner {
+
+    public static String[][] createscedule() {
+        Scanner sc = new Scanner(System.in);
+
+        String[][] schedule = {{"Monday", ""}, {"Tuesday", ""}, {"Wednesday", ""}, {"Thursday", ""}, {"Friday", ""}, {"Saturday", ""}, {"Sunday", ""}};
+
+        for (int i = 0; i < schedule.length; i++) {
+            System.out.print("Enter the task for " + schedule[i][0] + ": ");
+            schedule[i][1] = sc.nextLine();
+        }
+
+        System.out.println("Schedule is created!");
+        return schedule;
+    }
+
+    public static String[][] reschedule(String[][] schedule) {
+        Scanner sc = new Scanner(System.in);
+
+        while (true) {
+            System.out.print("Enter the day of the week to change: ");
+            String day = sc.nextLine().trim();
+
+            for (int i = 0; i < schedule.length; i++) {
+                if (schedule[i][0].equalsIgnoreCase(day)) {
+                    System.out.print("Enter the task: ");
+                    schedule[i][1] = sc.nextLine();
+                    System.out.println("Schedule is changed!");
+                    return schedule;
+                }
+            }
+            System.out.println("Input is incorrect, please, try again!");
+        }
+    }
+
+
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Hello, please create your schedule first!");
+        String[][] schedule = createscedule();
+
+
+        while (true) {
+            System.out.println("Please, enter: \n 1. Day of the week to see schedule \n 2. 'Reschedule' to change task list \n 3. 'Exit' to exit from the program ");
```

Comment:
pls remove this menu, in documentation said
**_Improve the program so that when the user enters the phrase change [day of the week] or reschedule [day of the week], the program offers to enter new tasks for a given day of the week and saves them in the appropriate cell of the array, while removing the old tasks._**